### PR TITLE
add Arbitrum (mainnet+sepolia) addresses

### PR DESCRIPTION
**Pull request by scott-silver**

### CODE REVIEW

File: src/builder/BridgeRoutes.sol [hunk 1]
```diff
@@ -36,6 +37,8 @@ library CCTP {
         chains[2] = CCTPChain({chainId: 11155111, domainId: 0, bridge: 0x9f3B8679c73C2Fef8b59B4f3444d4e156fb70AA5});
         // Base Sepolia
         chains[3] = CCTPChain({chainId: 84532, domainId: 6, bridge: 0x9f3B8679c73C2Fef8b59B4f3444d4e156fb70AA5});
+        // Arbitrum Sepolia
+        chains[4] = CCTPChain({chainId: 421614, domainId: 3, bridge: 0x9f3B8679c73C2Fef8b59B4f3444d4e156fb70AA5});
```
Comment (scott-silver) [inline src/builder/BridgeRoutes.sol:41]: source: https://developers.circle.com/stablecoins/evm-smart-contracts

File: src/builder/UniswapRouter.sol [hunk 2]
```diff
@@ -21,6 +21,8 @@ library UniswapRouter {
         chains[2] = RouterChain({chainId: 11155111, router: 0x3bFA4769FB09eefC5a80d6E87c3B9C650f7Ae48E});
         // Base Sepolia
         chains[3] = RouterChain({chainId: 84532, router: 0x94cC0AaC535CCDB3C01d6787D6413C739ae12bc4});
+        // Arbitrum Sepolia
+        chains[4] = RouterChain({chainId: 421614, router: 0x101F443B4d1b059569D643917553c771E1b9663E});
```
Comment (scott-silver) [inline src/builder/UniswapRouter.sol:25]: source: https://docs.uniswap.org/contracts/v3/reference/deployments/arbitrum-deployments

**Comment by scott-silver** [comment]
Not sure if we need to update this contract as well (for Arbitrum Sepolia):

https://github.com/compound-finance/quark-scripts/blob/main/src/lib/UniswapFactoryAddress.sol

**Comment by kevincheng96** [comment]
> Not sure if we need to update this contract as well (for Arbitrum Sepolia):
> 
> https://github.com/compound-finance/quark-scripts/blob/main/src/lib/UniswapFactoryAddress.sol

Probably would be a good idea to do that as well. Though none of the contracts that currently use that library are deployed and being used right now.

**Review by kevincheng96 — APPROVED** [review]
Double checked the addresses and they all look good 👍

**Review by coburncoburn — APPROVED** [review]

**Pull request merged**